curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/663b10c10a4c0a0e095137ee.md

```diff
@@ -0,0 +1,54 @@
+---
+id: 663b10c10a4c0a0e095137ee
+title: Step 18
+challengeType: 0
+dashedName: step-18
+---
+
+# --description--
+
+After returning the list `snake_cased_char_list`, you will need to join its elements into a single string using an empty string `''` as the separator.
```

Comment:
```suggestion
Instead of returning the list `snake_cased_char_list`, you will need to join its elements into a single string using an empty string `''` as the separator.
```